### DISCUSSION TITLE
What is the equivalent of Chromium's Ctrl+Shift+R?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#6998] [Q&A] [answered]
When I had issues with a web page in my company domain, I was suggested to refresh via Ctrl+Shift+R, and they obviously referred to Chrome. It worked on Chromium, but I would I do the same in qutebrowser?

**Comment by The-Compiler** [comment, marked as answer]
`Shift-R` is bound to `:reload --force`, have you tried that?

**Reply by Aster89** [reply]
Nope. I guess it must be it. Will try next time I needed. By the way, I see that also `R` is bound to the same. I should have guess it!

**Reply by The-Compiler** [reply]
> Nope. I guess it must be it. Will try next time I needed. By the way, I see that also `R` is bound to the same. I should have guess it!

R is Shift-R.

**Reply by Aster89** [reply]
Ok, apparently my fingers hit Shift-R, and then I thought it was Ctrl-R or something. Then I ri-discovered that Shift-R worked too, and I was suprised too. Sorry, my memory blacks out every now and then :/
